prow/scripts/cluster-integration/provision-gke-cluster.sh

```diff
@@ -32,7 +33,10 @@ if [ "${discoverUnsetVar}" = true ] ; then
     exit 1
 fi
 
-CLUSTER_VERSION_PARAM="--cluster-version=1.10.7"
+DEFAULT_VERSION="1.10.7"
```

Comment:
it is only smarter to what we have now if we will stick to "1.10" as a default version. Question is do we want it, and if there's any risk in it. @aszecowka what do you think? Also we would need to update docu on kyma side (over there we stick to "1.10.7")